meta_request/lib/meta_request/log_interceptor.rb

```diff
@@ -3,33 +3,33 @@
 module MetaRequest
   module LogInterceptor
 
-    def debug(message = nil, &block)
-      push_event(:debug, message)
+    def debug(*message)
+      push_event(:debug, message.join(' '))
```

Comment:
Yeah, that works too.  I put the `join` in case it was desirable to do "something" with all the extra arguments in `push_event`.  I removed the `&block` argument because it's not needed and it slows down execution.

I'll make another commit with `message=nil, *args`.  I can add `&block` back, too, but it does slow things down (from 22% to over 500% more overhead on Ruby 2.3.0):

```ruby
def foo; yield nil if block_given?; end
def bar(&block); yield nil if block_given?; end

Benchmark.realtime do 100000000.times do fooy end end
# => 6.277454078197479
Benchmark.realtime do 100000000.times do bary end end
# => 7.684779674280435
Benchmark.realtime do 100000000.times do fooy{} end end
# => 8.721442708745599
Benchmark.realtime do 100000000.times do bary{} end end
# => 47.150375722907484
```